src/Marten.Testing/Internal/Generated/DocumentStorage/AccountProvider793732554.cs

```diff
@@ -1,878 +0,0 @@
-// <auto-generated/>
-#pragma warning disable
-using Marten.Internal;
-using Marten.Internal.Storage;
-using Marten.Schema;
-using Marten.Schema.Arguments;
-using Marten.Testing.Documents;
-using Npgsql;
-using System;
-using System.Collections.Generic;
-using Weasel.Core;
-using Weasel.Postgresql;
-
-namespace Marten.Generated.DocumentStorage
```

Comment:
Should those generated files be removed?

Reply:
It'd speed up the tests if we left them in place